### PR TITLE
Delete button glitch fixed, styled table, FINISHED(?) authentication

### PR DESCRIPTION
**Pull request by eryoon**
I accidentally used "Heading", a nonexistent component, instead of <Header>..

### CODE REVIEW

**Comment by eryoon** [comment]
Remember to `npm install` again!
Also, I moved the rendering to a func in actions.js, because we'll need to render again after the user successfully signs in.

**Comment by eryoon** [comment]
Of course, we replace the `var signedIn = false` with the state of our login. Probably just if a cookie is null.

Also, can someone figure out how to space the elements in the signin page? It's pretty ugly.

File: src/containers/CalendarContainer.js [hunk 1]
```diff
@@ -18,22 +18,39 @@ export default class CalendarContainer extends React.Component {
...
+            }).catch(function(error) {
+            var errorCode = error.code;
+            var errorMessage = error.message;
+            var email = error.email;
+            var credential = error.credential;
+
+            // ************************************************************
+            // *DELETE THIS CONSOLE.LOG WHEN DONE TO AVOID SECURITY FLAWS!*
+            // ************************************************************
+            console.log("We couldnt log in! Err: " + errorCode + ", errmsg: " + errorMessage + ", email: " + email + ", cred: " + credential);
```
Comment (mirman-mtaggart) [inline src/containers/CalendarContainer.js]: Fun fact: there is a `console.error()` for errors.

**Comment by eryoon** [comment]
Still getting the "you need to whitelist" error

**Comment by mirman-mtaggart** [comment]
Does [this](https://github.com/firebase/quickstart-js/blob/master/auth/chromextension/credentials.js) help? Looks like the "right" way to do this with a Chrome extension.

**Comment by eryoon** [comment]
OK, I found a new solution. You can get the [google credential from chrome](https://developer.chrome.com/apps/app_identity) directly.
[Here's a helpful link](https://github.com/firebase/quickstart-js/tree/master/auth/chromextension)

**Comment by eryoon** [comment]
@mirman-mtaggart Finally finished up auth! Spent 9 hours (literally) on it.
...and 3 of those hours were spent for an error, when the solution was to type "null, "...

**Pull request merged**